aten/src/ATen/dlpack.h

```diff
@@ -132,10 +183,11 @@ typedef struct DLManagedTensor {
   /*! \brief Destructor signature void (*)(void*) - this should be called
    *   to destruct manager_ctx which holds the DLManagedTensor. It can be NULL
    *   if there is no way for the caller to provide a reasonable destructor.
+   *   The destructors deletes the argument self as well.
    */
   void (*deleter)(struct DLManagedTensor * self);
 } DLManagedTensor;
 #ifdef __cplusplus
 }  // DLPACK_EXTERN_C
 #endif
-#endif  // DLPACK_DLPACK_H_
+#endif  // DLPACK_DLPACK_H_:
```

Comment:
This spurious `:` is the only delta with https://github.com/dmlc/dlpack/blob/main/include/dlpack/dlpack.h - probably a typo?

Reply:
yes! i do :wq poorly sometimes 😅

Reply:
fellow Vim user here, I get it :)